js/_1broker.js

```diff
@@ -10,36 +10,36 @@ const { ExchangeError, NotSupported } = require ('./base/errors');
 module.exports = class _1broker extends Exchange {
     describe () {
         return this.deepExtend (super.describe (), {
-            'id': '_1broker',
-            'name': '1Broker',
-            'countries': [ 'US' ],
-            'rateLimit': 1500,
-            'version': 'v2',
-            'has': {
-                'publicAPI': false,
-                'CORS': true,
-                'fetchTrades': false,
-                'fetchOHLCV': true,
+            id: '_1broker',
+            name: '1Broker',
+            countries: ['US'],
+            ratelImit: 1500,
+            version: 'v2',
+            has: {
+                publicaPI: false,
```

Comment:
↑ oops ) 

Btw, you would make it much easier for me to merge if you could split this into several atomic PRs, otherwise it will quickly run into a conflict with ongoing changes.

Thx anyways!

Reply:
Fokken python regex))

Reply:
It happens, no worries )